<ui version="4.0" >
 <author></author>
 <comment></comment>
 <exportmacro></exportmacro>
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>428</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QGLViewer + Qt4</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QVBoxLayout" >
    <property name="margin" >
     <number>9</number>
    </property>
    <property name="spacing" >
     <number>6</number>
    </property>
    <item>
     <widget class="MyQGLViewer" name="viewer" />
    </item>
    <item>
     <layout class="QHBoxLayout" >
      <property name="margin" >
       <number>0</number>
      </property>
      <property name="spacing" >
       <number>6</number>
      </property>
      <item>
       <widget class="QCheckBox" name="checkBox" >
        <property name="text" >
         <string>FPS</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_2" >
        <property name="text" >
         <string>Grid</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label" >
        <property name="text" >
         <string>Radius :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="radiusSpinBox" >
        <property name="prefix" >
         <string/>
        </property>
        <property name="maximum" >
         <double>5</double>
        </property>
        <property name="minimum" >
         <double>0.1</double>
        </property>
        <property name="singleStep" >
         <double>0.5</double>
        </property>
       </widget>
      </item>
      <item>
       <spacer>
        <property name="orientation" >
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" >
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="pushButton" >
        <property name="text" >
         <string>Quit</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>29</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuFile" >
    <property name="title" >
     <string>File</string>
    </property>
    <addaction name="actionOpen" />
    <addaction name="actionQuit" />
   </widget>
   <addaction name="menuFile" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>406</y>
     <width>594</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpen" >
   <property name="text" >
    <string>Open</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About</string>
   </property>
  </action>
 </widget>
 <pixmapfunction></pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>MyQGLViewer</class>
   <extends>QWidget</extends>
   <header>my_qglviewer.h</header>
   <container>1</container>
   <pixmap></pixmap>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>536</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>535</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox</sender>
   <signal>clicked(bool)</signal>
   <receiver>viewer</receiver>
   <slot>setFPSIsDisplayed(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>55</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>61</x>
     <y>344</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewer</sender>
   <signal>FPSIsDisplayedChanged(bool)</signal>
   <receiver>checkBox</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>42</x>
     <y>332</y>
    </hint>
    <hint type="destinationlabel" >
     <x>39</x>
     <y>384</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBox_2</sender>
   <signal>clicked(bool)</signal>
   <receiver>viewer</receiver>
   <slot>setGridIsDrawn(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>109</x>
     <y>392</y>
    </hint>
    <hint type="destinationlabel" >
     <x>160</x>
     <y>353</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>viewer</sender>
   <signal>gridIsDrawnChanged(bool)</signal>
   <receiver>checkBox_2</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>126</x>
     <y>347</y>
    </hint>
    <hint type="destinationlabel" >
     <x>109</x>
     <y>392</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
